src/games/strategy/triplea/ai/fastAI/FastOddsEstimator.java

```diff
@@ -0,0 +1,146 @@
+package games.strategy.triplea.ai.fastAI;
```

Comment:
How exactly, how much, and why is this calculator faster than the normal one? The usual calculator is dominated performance wise due to the 'clone data' call. Seems like the speed up here is because 'setGameData' does not do the clone. Can we get rid of the clone in the main calculator?